docs/index.rst

```diff
@@ -0,0 +1,46 @@
+Jupyter Docker Stacks
+=====================
+
+Jupyter Docker Stacks are a set of ready-to-run Docker images containing Jupyter applications and interactive computing tools. You can use a stack image to start a personal Jupyter Notebook server in a local Docker container, to run JupyterLab servers for a team using JupyterHub, to write your own project Dockerfile, and so on.
```

Comment:
thought: make this a bulleted list with links to *how* to do each of these things (if that makes sense)